odxtools/structures.py

```diff
@@ -224,30 +224,27 @@ def convert_physical_to_bytes(self,
             is_end_of_pdu=encode_state.is_end_of_pdu,
         )
 
-    def convert_bytes_to_physical(self, decode_state: DecodeState, bit_position: int = 0):
+    def convert_bytes_to_physical(self,
+                                  decode_state: DecodeState,
+                                  bit_position: int = 0) -> Tuple[ParameterValueDict, int]:
```

Comment:
maybe make an alias for `Tuple[ParameterValueDict, int] `?

Reply:
not sure: this is pretty deep in the decoding logic and it is easier to see what's returned if this is directly specified. 

That said, I think the decoding and encoding code would benefit from a major overhaul. In this case, it would possibly be better if the position of the next byte would be put into `decode_state`. (but maybe not, because this would make the arguments non-const...)